network/tests/test_linux.py

```diff
@@ -204,6 +208,9 @@ def test_collect_cx_queues_when_ss_fails(check, aggregator):
     for metric in CONNECTION_QUEUES_METRICS + common.EXPECTED_METRICS + common.EXPECTED_WINDOWS_LINUX_METRICS:
         aggregator.assert_metric(metric)
 
+    # TODO Add this assert back when `assert_metrics_using_metadata` properly handles histograms
```

Comment:
Do we have a task for this?

Reply:
> Additional Notes
I decided to rename the metrics in the metadata file instead of renaming them in the check itself to avoid a breaking change. If you think we should do the other way around, let me know.
I'll sync the metadata once merged
About [this](https://github.com/DataDog/integrations-core/pull/13937/files#diff-9c36a94ac8c1cb8c65b5d7f5e3b85250eb2a409994d962a9a7989066b8e67ba7R194-R195) and [this](https://github.com/DataDog/integrations-core/pull/13937/files#diff-9c36a94ac8c1cb8c65b5d7f5e3b85250eb2a409994d962a9a7989066b8e67ba7R211-R212), I created https://datadoghq.atlassian.net/browse/AITOOLS-131